x-pack/plugins/security_solution_serverless/public/upselling/pages/endpoint_exceptions_details.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiEmptyPrompt, EuiIcon } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import React, { memo } from 'react';
+import type { AppFeatureKeyType } from '@kbn/security-solution-features/keys';
+import { getProductTypeByPLI } from '../hooks/use_product_type_by_pli';
+
+const EndpointExceptionsDetailsUpselling: React.FC<{ requiredPLI: AppFeatureKeyType }> = memo(
+  ({ requiredPLI }) => {
+    const productTypeRequired = getProductTypeByPLI(requiredPLI);
+
+    return (
+      <EuiEmptyPrompt
+        icon={<EuiIcon type="logoSecurity" size="xl" />}
+        color="subdued"
+        title={
+          <h2>
+            <FormattedMessage
+              id="xpack.securitySolutionServerless.endpoint.exceptions.details.paywall.title"
+              defaultMessage="Do more with Security!"
+            />
+          </h2>
+        }
+        body={
+          <p>
+            <FormattedMessage
+              id="xpack.securitySolutionServerless.endpoint.exceptions.details.paywall.body"
+              defaultMessage="Upgrade your license to {productTypeRequired} to use Endpoint Security Exception List."
```

Comment:
@caitlinbetz @kevinlog The info message needs an update here.

Reply:
Will we (our team) continue to be the ones responsible for these files? if so, can I ask that you create a folder (maybe name it the same as the one I created under `sections/` - `endpoint_management`) and update the `CODEOWNERS` file to show our team as owners of it?